sprites/bug/bugCharacter.js

```diff
@@ -1,14 +1,22 @@
 
 const bugCharacter = {
   name:"bug",
-
+  animationTypes: ['GREEN','BLUE','RED','YELLOW', 'ALL'],
```

Comment:
This is fine for now. I am still considering that it may be better to make this different sprite characters. Currently the idea of a sprite is that it is a character and its animations. This breaks that, in that these are really different characters.